test/zippy/grafana/datasources/prometheus.yml

```diff
@@ -0,0 +1,14 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+apiVersion: 1
+datasources:
+  - name: Prometheus
+    type: prometheus
+    url: http://prometheus:9090
```

Comment:
This doesn't seem like the right place for this file. Can we put it in `misc/mzcompose` instead, so that it can be reused by other workflows that want to use Prometheus/Grafana?